src/app/_components/common/H6.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { type CSSProperties } from "react";
+import { cn } from "~/app/lib/utils";
+
+interface Props {
+  children: React.ReactNode;
+  className?: string;
+  style?: CSSProperties;
+}
+export const H6: React.FC<Props> = ({ children, className, style }: Props) => {
+  return (
+    <>
+      <h3 style={style} className={cn("text-xl uppercase", className)}>
+        {children}
+      </h3>
+    </>
+  );
+};
```

Comment:
We should have a blank line between the interface and the function definition
```suggestion
interface Props {
  children: React.ReactNode;
  className?: string;
  style?: CSSProperties;
}

export const H6: React.FC<Props> = ({ children, className, style }: Props) => {
  return (
    <>
      <h3 style={style} className={cn("text-xl uppercase", className)}>
        {children}
      </h3>
    </>
  );
};
```